### PR TITLE
951 badge icon

### PR DESCRIPTION
**Pull request by Radha-Alberta**

### CODE REVIEW

**Review by chrisolsen — DISMISSED** [review]
Can you squash these commits?

File: libs/docs/src/components/common/Badge.stories.mdx [hunk 1]
```diff
@@ -166,7 +166,7 @@ which display minimal amounts of information, system feedback, or states.
     <CodeSnippet
       lang="html"
       code={`
-        <goa-badge type="information" content="Information" icon />
+        <goa-badge type="information" content="Information" icon `/>
```
Comment (chrisolsen) [inline libs/docs/src/components/common/Badge.stories.mdx:169]: Looks like a backtick was accidentally added to this line

**Comment by chrisolsen** [comment]
These changes seem to be doing the opposite of what is specified in the ticket, which is to convert `icon` to `icon="true"`. Now, I believe both are valid on the Angular side, but I would want to verify that just `icon` converts properly to the web component

https://goa-dio.atlassian.net/jira/software/c/projects/DDIDS/boards/185?modal=detail&selectedIssue=DDIDS-951

**Comment by ArakTaiRoth** [comment]
> Can you squash these commits?

@chrisolsen I can squash these when I merge them in, not a big deal.

**Comment by ArakTaiRoth** [comment]
> These changes seem to be doing the opposite of what is specified in the ticket, which is to convert `icon` to `icon="true"`. Now, I believe both are valid on the Angular side, but I would want to verify that just `icon` converts properly to the web component
> 
> https://goa-dio.atlassian.net/jira/software/c/projects/DDIDS/boards/185?modal=detail&selectedIssue=DDIDS-951

I told her to go this way with reducing it to just `icon`. I thought that was a simpler approach, and the whole intent of the story was to make things the same throughout. But that's fair, we should probably test to ensure that this way works in both React and Angular.

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**